app/Models/Thread.php

```diff
@@ -146,4 +164,15 @@ public static function feedQuery(): Builder
                 END AS latest_creation
             '));
     }
+
+    /**
+     * retrieve 20 feed items with use of page parameter.
+     */
+    public static function getFeedItems()
+    {
+        $page = intval(request('page', 1));
+        $query = static::feedQuery();
+
+        return $query->skip(($page - 1) * static::FEED_PAGE_SIZE)->take(static::FEED_PAGE_SIZE)->get();
```

Comment:
It is possible to use built-in Laravel pagination here?

https://github.com/laravel/framework/blob/5.6/src/Illuminate/Database/Eloquent/Builder.php#L738

Reply:
@joedixon is this possible? ^